hasura/migrations/default/1676345211557_org_members/up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE public.org_members (
+    id BIGSERIAL PRIMARY KEY,
+    profile_id bigint NOT NULL REFERENCES profiles(id) ON DELETE RESTRICT ON UPDATE RESTRICT,
+    org_id bigint NOT NULL REFERENCES organizations(id) ON DELETE RESTRICT ON UPDATE RESTRICT,
+    deleted_at timestamp without time zone,
+    created_at timestamp without time zone NOT NULL DEFAULT now(),
+    updated_at timestamp without time zone,
```

Comment:
@levity can you add a trigger to this field to update on row changes, as we do with all other tables.